packages/opentelemetry-metrics/test/Meter.test.ts

```diff
@@ -48,7 +48,7 @@ describe('Meter', () => {
 
     describe('.getHandle()', () => {
       it('should create a counter handle', () => {
-        const counter = meter.createCounter('name');
+        const counter = meter.createCounter('name') as CounterMetric;
```

Comment:
Just curious, why do you need to use `as` - Type assertion here? Earlier it used to work, Is it because we are returning noop implementation?

Reply:
Yes, the function `createCounter` now returns `types.Metric<types.CounterHandle>` where it used to return just `CounterMetric`. The return type needed to be changed in order to allow returning the noop implementation which satisfies the type types.Metric<types.CounterHandle> but is not a `CounterMetric`

Reply:
Got you. As a user, I have to use/do the same assertion? This looks little inconvenient. WDYT?

Reply:
No you don't, because users will only use the public interface. This is only done because the tests access private fields